### ISSUE TITLE
Simple tutorial on DisplayPosts.com

### ISSUE THREAD

**Issue by billerickson** [shazahm1/Display-Posts-Shortcode-Remote#4]
I received an email asking for guidance on using this plugin (hence the recent GH issues :) ).

![screenshot](https://d3vv6lp55qjaqc.cloudfront.net/items/3X011i3Z1v0c1X1U2r3E/Screen%20Shot%202019-09-16%20at%2010.38.31%20AM.png?X-CloudApp-Visitor-Id=57dd62b2e547854fee104e672ecd4cfa&v=54c9c1e4)

Would you be interested in writing a short tutorial on the [DisplayPosts.com](https://displayposts.com) website outlining how you use this plugin? I think a bit of documentation would really help users who are interested in using this.

Thanks

**Comment by shazahm1** [comment]
Yes, the same user emailed me too. I'll add a readme.md to the repo this week. You can either include a link or copy/paste it on DisplayPosts.com if you wish. I am fine with either.

**Comment by UudetYr** [comment]
Hi,
Is this tutorial available?
This remote extension would be amazing for my website. Please help me to understand how to use it.
